web/vtctld2/src/app/dashboard/shard.component.ts

```diff
@@ -72,20 +65,55 @@ export class ShardComponent implements OnInit, OnDestroy {
     });
   }
 
+  getActions() {
+    let result: MenuItem[] = [
+      {label: 'Status', items: [
+        {label: 'Validate', command: (event) => {this.openValidateShardDialog(); }},
+        {label: 'Validate Version', command: (event) => {this.openValidateVerShardDialog(); }},
+        {label: 'Shard Replication Positions', command: (event) => {this.openShardReplicationPosDialog(); }},
+      ]},
+      {label: 'Change', items: [
+        {label: 'Delete', command: (event) => {this.openDeleteShardDialog(); }},
+        {label: 'Tablet Externally Reparented', command: (event) => {this.openTabExtRepDialog(); }},
```

Comment:
I think it's very confusing to have "Tablet Externally Reparented" item on a shard. Why it's not on a tablet?